src/phoenix/experimental/evals/evaluators.py

```diff
@@ -1,16 +1,19 @@
 from typing import List, Mapping, Optional, Tuple
 
 from phoenix.experimental.evals.models import set_verbosity
-from phoenix.experimental.evals.utils import parse_openai_function_call, snap_to_rail
+from phoenix.experimental.evals.utils import (
+    NOT_PARSABLE,
+    openai_function_call_kwargs,
+    parse_openai_function_call,
+    snap_to_rail,
+)
 from phoenix.utilities.logging import printif
 
-from .models import BaseEvalModel
+from .models import BaseEvalModel, OpenAIModel
```

Comment:
isn't this gonna automatically import openAI and fail if the user doesn't have openAI?

Reply:
Should be okay since we dynamically import the OpenAI library upon instantiation of the `OpenAIModel` instance. https://github.com/Arize-ai/phoenix/blob/51435297538c83542f7df6ba5860a50ef71307cf/src/phoenix/experimental/evals/models/openai.py#L111

Reply:
I'm testing...

Reply:
This script executes successfully in an environment without `openai`.

```python
from phoenix.experimental.evals import OpenAIModel

error_on_instantiation = False
try:
    model = OpenAIModel()
except ImportError:
    error_on_instantiation = True

assert error_on_instantiation
```